DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UIHostingViewRecorder.swift

```diff
@@ -186,6 +186,40 @@ internal struct UIHostingWireframesBuilder: NodeWireframesBuilder {
                 cornerRadius: info?.cornerRadius,
                 opacity: info?.alpha
             )
+        case let .image(graphics):
+            return graphics.contents.map { contents in
+                builder.createImageWireframe(
+                    id: Int64(content.seed.value),
+                    resource: UIImageResource(
+                        image: UIImage(
+                            cgImage: contents.cgImage,
+                            scale: graphics.scale,
+                            orientation: {
```

Comment:
nit: we could extract the `graphics.orientation` switch statement into a separate function or computed property for cleaner code.

Probably as well on `maskColor` used below.

Reply:
Yes, let me change that 👍